client/my-sites/email/email-management/home/email-plan-warnings.jsx

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { CompactCard } from '@automattic/components';
+
+const EmailPlanWarnings = ( { warnings } ) =>
+	warnings.map( ( warning, index ) => (
```

Comment:
At a slightly higher level, might it make sense to use a `Notice` component instead of a `CompactCard`? I am wary of adding an embedded card here - I'd lean towards a Notice or using a simpler `<div>` that is styled to have an `(i)` icon followed by a warning message and an optional action. (This is what the Notice is, pretty much, with the exception that it has fairly opinionated styles!)

Reply:
A `Notice` sounds good